actionpack/test/controller/redirect_test.rb

```diff
@@ -601,6 +601,23 @@ def test_url_from_fallback
     end
   end
 
+  def test_redirect_to_instrumentation
+    payload = nil
+
+    subscriber = proc do |*args|
+      event = ActiveSupport::Notifications::Event.new(*args)
```

Comment:
It's minor but in https://github.com/rails/rails/pull/49433 I was hoping people would move away from this pattern ;)

Reply:
Bedankt @bdewater  😄  https://github.com/rails/rails/pull/49467